docker/caffe2/jenkins/common/install_rocm.sh

```diff
@@ -42,19 +46,55 @@ install_ubuntu() {
 }
 
 install_centos() {
-    echo "Not implemented yet"
-    exit 1
+
+  yum update -y
+  yum install -y wget
+  yum install -y openblas-devel
+
+  yum install -y centos-release-scl
+  yum install -y devtoolset-7
```

Comment:
can we put this to a separate install_devtoolset.sh? and then add "-devtoolset7-" in our docker image name.

Reply:
Sure.